README.md

```diff
@@ -91,8 +91,14 @@ Then the shanoir-downloader project can be simply managed as a normal git repo (
 * Fork GitHub project
 * Clone
 * Edit .env file and set SHANOIR_MIGRATION=init
+
+* Using locally built docker images
 * Run "mvn clean install -DskipTests" in shanoir-ng/shanoir-ng-parent/ folder
+* Run docker-compose up -f docker-compose-dev.yml --build in shanoir-ng/
```

Comment:
Hi @jcomedouteau, thx for this PR. On my machine I have to put the -f before the up, that it works. Are you sure with your command?

Reply:
You're totally right ! That's corrected